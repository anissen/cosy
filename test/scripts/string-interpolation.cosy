print '4 * 5 = ${4 * 5}'
print '7.5 / 3 = ${7.5 / 3}'
print '${1}'
print '|${1}|'
print '${1}${2}'
print '|${1}|${2}|'
print '|${'hello from the inside'}|'

fn square(v Num) {
    return v * v
}
print '5 squared is ${square(5)}'

struct Test {
    var value Str
}
var t = Test { value = 'hello' }
print 'Test struct: ${t}'
