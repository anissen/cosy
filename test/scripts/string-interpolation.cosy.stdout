4 * 5 = 20
7.5 / 3 = 2.5
1
|1|
12
|1|2|
|hello from the inside|
5 squared is 25
Test struct: Test { value = "hello" }
